PyRDF/Operation.py

```diff
@@ -88,42 +89,8 @@ def __init__(self, name, *args, **kwargs):
     def _classify_operation(self, name):
         # Classifies the given operation as action or
         # transformation and returns the type.
-
-        future_ops = [
```

Comment:
`future_ops` can perhaps go away from `Operation`, and be part of the check for the support in the corresponding `Backend`. Or even disappear completely (we just make a distinction between supported and not supported, we do not talk about future plans).